analyzers/tests/SonarAnalyzer.UnitTest/Helpers/PropertiesHelperTest.cs

```diff
@@ -51,15 +53,14 @@ public class PropertiesHelperTest
         public void ShouldAnalyzeGeneratedCode_WithNotBooleanValue_ReturnsFalse() =>
             GetSetting("ResourceTests\\NotBoolean\\SonarLint.xml").Should().BeFalse();
 
-        [TestMethod]
-        public void ShouldAnalyzeGeneratedCode_WithNotExistingPath_ReturnsFalse() =>
-            GetSetting("ResourceTests\\NotExistingFolder\\SonarLint.xml").Should().BeFalse();
-
         private static bool GetSetting(string filePath)
         {
             // Arrange
+            var content = SourceText.From(File.ReadAllText(filePath));
```

Comment:
All the tests of this class are now working in-memory additional file but receiving their data from the disk prep files for the sake of minimal changes.